src/assets/stylesheets/shared/_layout.scss

```diff
@@ -237,8 +237,7 @@ footer
 }
 
 .btn{
-  min-width: 90px;
-  font-size: $font-size-base;
```

Comment:
why remove these? is the default font-size of every button throughout the environment still going to be at 14px with the removal of this?